### PR TITLE
kubeadm: update OWNERS for 1.21

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:

as part of the annual SIG reports we should keep OWNERS files up-to-date:
https://groups.google.com/g/kubernetes-sig-cluster-lifecycle/c/KH8g4WRjOAE

notes:
- if you object to the bellow amends to these OWNERs files just state so, or give your +1 / -1.
- applying a lazy consensus deadline mid next week / ~4th of Feb.
- the file under /test/ is just a copy of the one from /cmd/kubeadm
- duplicating approvers under reviewers is not needed.

removals:
- @luxas: Lucas is an emeritus SIG chair and is currently in university. we can re-add him once / if he comes back to kubeadm.
- @timothysc: Tim is a SIG chair, but has been busy with downstream work and is mostly active in the main SIG process.
- @rosti: has stated that he will no longer work on kubeadm, due to downstream commitments in other areas.
- @ereslibre: has stopped contributing, likely to downstream commitments or other reasons.
- @detiber, @dixudx, @yastij, @kad are mostly inactive in reviews, so we'd like to give the bot pings to the active team.

thank you all for your contributions and hope to see you again!

additions / promotions:
- @SataQiu: has been with kubeadm for a number of releases, being active in both review and fixing critical problems. suggesting a promotion to approver.
- @pacoxu: has been quite active in a lot of kubeadm PRs / issues recently. suggesting a promotion to reviewer.

**Which issue(s) this PR fixes**:

xref https://groups.google.com/g/kubernetes-sig-cluster-lifecycle/c/KH8g4WRjOAE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by ereslibre** [comment]
+1 from my side. Thank you for everything during this time, learned a lot and had lots of fun :pray:

**Comment by kad** [comment]
Sorry about my side. I'm lately more busy with resource management topics and device, but will try to come back to kubeadm soon.

File: cmd/kubeadm/OWNERS
Comment (detiber) [inline cmd/kubeadm/OWNERS:5]: nit: removals from approvers should probably added to `emeritus_approvers`: https://github.com/kubernetes/community/blob/master/contributors/guide/owners.md#emeritus

**Review by detiber — COMMENTED** [review]
+1 from me for the proposed role changes, added a comment with a nit that former approvers should likely be added to `emeritus_approvers` rather than fully removed.

File: cmd/kubeadm/OWNERS
Comment (neolit123) [reply cmd/kubeadm/OWNERS:5]: thanks, i remembered about `emeritus_approvers` right after sending the PR.

**Comment by fabriziopandini** [comment]
+1 for everything, including the emeritus approver section
thanks to everyone for the great work!

**Comment by yastij** [comment]
lgtm

**Review by dixudx — COMMENTED** [review]
+1 from my side.

**Comment by neolit123** [comment]
update to include the `emeritus_approvers` section.

**Comment by fabriziopandini** [comment]
sgtm let's give some more time for people to review this

**Review by luxas — APPROVED** [review]
LGTM, thanks all for your continued stewardship of kubeadm :100:

(Now I'm in 2nd year of 5 in Uni now, things are going well :smile:, but it is a lot to cover :book:; so still don't have any extra time over unfortunately)

**Comment by neolit123** [comment]
thanks @luxas, best of luck in the remaining years at university!

**Comment by neolit123** [comment]
/hold cancel
/cc @fabriziopandini

looking for LGTM.

**Comment by pacoxu** [comment]
/retest
#98893

**Pull request merged**
